Python/Python.sublime-syntax

```diff
@@ -1034,7 +1034,7 @@ contexts:
         2: storage.modifier.conversion.python
         3: constant.other.format-spec.python
         4: punctuation.definition.placeholder.end.python
-    - match: \{(?=[^\}]+\{) # complex (nested) form
+    - match: \{(?=[^\}"']+\{) # complex (nested) form
```

Comment:
Does this break `"aaaa{'foo"`? I don't know why someone would do that, but I think we should add tests for it.